### DISCUSSION TITLE
Dumping ordered_json takes a very long time compared to json

### DISCUSSION THREAD

**Discussion by psiddire** [nlohmann/json#4442] [General]
I've used ordered_json's for maintaining the order of insertion into the JSON.
I'm dealing with JSONs which are huge. Over 100000 lines.
It takes 100x more time to dump ordered_json compared to json.
Is there a fix planned for this? Any workaround (apart from moving back to json :) )

**Comment by nlohmann** [comment]
Debug mode?

**Comment by psiddire** [comment]
I've narrowed down where the main issue is coming from.
I've the following line which is taking the most time.
Once the json is filled with data, I've 
`json_name.dump()`

Is this function not optimized for ordered_json? Any alternative?
